<?xml version="1.0"?>
<launch>
    <include file="$(find assignment_2_2022)/launch/assignment1.launch" />
    <param name="freq" type="double" value="1.0" />
    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" />
    <node pkg="tf2_web_republisher" type="tf2_web_republisher" name="tf2_web_republisher"/>
    <node pkg="assignment_2_2022" type="node_b" name="node_b" output="screen" />
    <node pkg="assignment_2_2022" type="node_c" name="node_c" output="screen" launch-prefix="xterm -e" />
</launch>
